pkg/kubelet/lifecycle/handlers.go

```diff
@@ -61,7 +61,7 @@ func (hr *HandlerRunner) Run(containerID kubecontainer.ContainerID, pod *api.Pod
 			msg    string
 		)
 		output := ioutils.WriteCloserWrapper(&buffer)
-		err := hr.commandRunner.ExecInContainer(containerID, handler.Exec.Command, nil, output, output, false, nil)
+		err := hr.commandRunner.ExecInContainer(containerID, handler.Exec.Command, nil, output, output, false, nil, 0)
```

Comment:
@timstclair quoting a comment from @ncdc on Sep 26:

> This is for running lifecycle hooks. There is no user input.

In any case, I added the TODO as you asked, so you can investigate that later.